Update nodejs.md (#11039)
DataDog/documentation: content/en/tracing/setup_overview/setup/nodejs.md

**Default**:  `false`<br>
Whether to enable capturing runtime metrics. Port `8125` (or configured with `dogstatsd.port`) must be opened on the Agent for UDP.

: **Environment Variable**: `DD_SERVICE_MAPPING`<br>
**Default**: `null`<br>
**Example**: `mysql:my-mysql-service-name-db,pg:my-pg-service-name-db`<br>

> Provide service names for each plugin. Accepts comma separated with or without spaces.

Author: ```suggestion
Provide service names for each plugin. Accepts comma separated values with or without spaces.
```

> Provide service names for each plugin. Accepts comma separated with or without spaces.

Reviewer: ```suggestion
Provide service names for each plugin. Accepts comma separated `plugin:service-name` pairs, with or without spaces.
```

> Provide service names for each plugin. Accepts comma separated values with or without spaces.

Author: ```suggestion
Provide service names for each plugin. Accepts comma separated `plugin:service-name` pairs, with or without spaces.
```

> Provide service names for each plugin. Accepts comma separated `plugin:service-name` pairs, with or without spaces.

Reviewer 2: ```suggestion
Provide service names for each plugin. Accepts comma separated pairs,`plugin:service-name`, with or without spaces.
```
